test/e2e/viperize.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
this is the same lib that is in kubernetes/test/e2e/viperconfig.go right? if so, can we just use it from there instead of forking it? I think we are getting v1.20.2 from the mod.go file you've updated and I see it here:
https://github.com/kubernetes/kubernetes/blob/release-1.20/test/e2e/viperconfig.go

Reply:
We can't, because the visibility of those functions changed from public to private.